apps/src/templates/FixZoomHelper.jsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import msg from '@cdo/locale';
```

Comment:
NIT: in general, we've been using `i18n` as the variable name for messages from the common locale file and `msg` as the variable name for app specific messages. Not a strict requirement - just a trend I've noticed.